src/esp/bindings/SimBindings.cpp

```diff
@@ -113,8 +115,12 @@ void initSimBindings(py::module& m) {
           &Simulator::setActiveSceneDatasetName,
           R"(The currently active dataset being used.  Will attempt to load
             configuration files specified if does not already exist.)")
-      /* --- Physics functions --- */
       /* --- Template Manager accessors --- */
+      .def_property(
+          "metadata_mediator", &Simulator::getMetadataMediator,
+          &Simulator::setMetadataMediator, py::return_value_policy::copy,
```

Comment:
Interesting, you really want to return a copy?

Reply:
copy of smart pointer.  want to increment holder count

Reply:
Please document in a comment.

Reply:
That's what happens by default for smart pointers with pybind

Reply:
That's what I thought but I wasn't sure. I'll remove the decorator then.